.vortex/CLAUDE.md

```
@@ -243,6 +278,20 @@ When script output changes, update corresponding test files:
 
 ### Common Test Commands
 
+**Documentation Workflow** (`.vortex/docs/`):
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)